webrtc.html

```diff
@@ -2871,6 +2929,242 @@
         </dl>
       </section>
 
+      <section>
+        <h3>RTCDtlsTransport Interface</h3>
+
+        <p>The <code><a>RTCDtlsTransport</a></code> interface allows an
+        application access to information about the Datagram Transport
+        Layer Security (DTLS) transport over which RTP and RTCP
+        packets are sent and received by <code><a>RTCRtpSender</a></code>
+        and <code><a>RTCRtpReceiver</a></code> objects, as well other data
+        such as SCTP packets sent and received by data channels.  In
+        particular, DTLS adds security to an underlying transport, and
+        the <code>RTCDtlsTransport</code> interface allows access to
+        information about the underlying transport and the security
+        added.
+
+        <dl class="idl" title=
+          "interface RTCDtlsTransport">
+          <dt>readonly attribute RTCIceTransport transport</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-dtlstransport-transport"><code>RTCDtlsTransport.transport</code></dfn>
+            attribute is the underlying transport that is used to send
+            and receive packets.  The underlying transport may not be
+            shared between multiple active <code><a>RTCDtlsTransport</a></code>
+            objects.
+          </dd>
+
+          <dt>readonly attribute RTCDtlsTransportState state</dt>
+
+          <dd>
+            <p>The <code id="dom-dtlstransport-state">state</code>
+            attribute MUST return the state of the transport.
+          </dd>
+
+          <dt>sequence&lt;ArrayBuffer&gt; getRemoteCertificates()</dt>
+
+          <dd>
+            <p>An array containing the remote certificates in use by
+            the remote side.</p>
+          </dd>
+
+          <dt>attribute EventHandler onstatechange</dt>
+
+          <dd>
+            This event handler, of event handler event
+            type <code><a href=
+            "#event-dtlstransport-statechange">statechange</a></code>,
+            MUST be fired any time the <a href=
+            "#dom-dtlstransport-state"><code><a>RTCDtlsTransport</a></code>
+            state</a> changes.
```

Comment:
This is exactly the same way it is written for PeerConnection.oniceconnectionstatechange and PeerConnection.onicegatheringstatechange and MediaStreamTrack.onisolationchange.